guide/lib/helpers/class_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module Helpers
+  module ClassHelpers
+    def class_names(*args, **kwargs)
```

Comment:
Make this a private method where it's needed instead? Quite hard to interpret what this does when looking at it out of context!

Reply:
This is only used within the guide, it's a crude reimplementation of Rails' `#class_names` - now the guide pulls in more of Rails, it might be possible to remove this altogether.

Reply:
Yeah, rebasing from `main` means this is no longer needed! Thanks for highlighting.